app/styles/component/_article-featured-video.scss

```diff
@@ -1,11 +1,13 @@
 .article-featured-video {
-	position: relative;
-	top: 0;
-	z-index: $z-site-head  + 1;
 
-	&.is-dismissed,
-	.slide-into-view + .page-wrapper & {
-		z-index: $z-site-head - 3;
+	&.on-scroll-variant {
+		position: relative;
+		z-index: $z-site-head  + 1;
+
+		&.is-dismissed,
+		.slide-into-view + .page-wrapper & {
```

Comment:
IMO parent selectors make this styling less readable as SCSS structure does not align with the DOM elements tree when we use them. What do you think?